www/src/examples/Navbar/Collapsible.js

```diff
@@ -1,7 +1,9 @@
 <Navbar collapseOnSelect expand="lg" bg="dark" variant="dark">
+  <Container>
   <Navbar.Brand href="#home">React-Bootstrap</Navbar.Brand>
   <Navbar.Toggle aria-controls="responsive-navbar-nav" />
   <Navbar.Collapse id="responsive-navbar-nav">
+    <Container>
```

Comment:
The `Container` under `Navbar.Collapse` isn't needed.

Reply:
Ack, got it. Removed container tag inside navbar collapse from collapsible.js. Should be good to go. Thank you so much for all your help @kyletsang